runner/sql.go

```diff
@@ -329,15 +334,21 @@ func importPanel(
 	panel panelToImport,
 	qt quoteType,
 	panelResultLoader func(string, string) (chan map[string]any, error),
+	cacheMode *bool,
```

Comment:
Using a pointer to a bool to represent tri-state is kind of sketchy looking.

Even if you need this inside dsq code, I don't think you need it here and you can just use regular bool.

Reply:
But I need something to flag importPanel to use temporary or normal table. How about using an enum int?

Reply:
Actually nevermind, I will just use an extra bool